src/lib/protocols/stun.c

```diff
@@ -293,7 +351,8 @@ int is_stun(struct ndpi_detection_module_struct *ndpi_struct,
                        flow->detected_protocol_stack[0]);
 
           if(1 /* TODO: enable/disable */ &&
```

Comment:
Is that still needed?
Was the idea to make this configurable?

Reply:
It was.. at the time... I'll remove the comment at the next iteration of this patch

Reply:
Done